docs/sdk-and-tools/elastic-search.md

```diff
@@ -19,6 +19,21 @@ Our implementation uses a concept of a queue and makes sure that everything is b
 
 ## Setup
 
+[comment]: # (mx-context-auto)
+
+### Option 1
+
+Set up four observers, each corresponding to a shard, with the `WebSocketHost` enabled, and create an instance of the `elasticindexer` service.
+Follow the instructions provided on [this](/sdk-and-tools/indexer) page to accomplish this.
+
+[comment]: # (mx-context-auto)
+
+### Option 2 
+
+:::warning
+this Option will be removed on the release `rc/v1.7.0`
```

Comment:
we are not really sure about this now, maybe state this as:
```
we plan to remove this option in the release `rc/v1.7.0`
```

Reply:
changed